src/components/20-molecules/accordion/index.js

```diff
@@ -5,11 +5,49 @@ import expandMore from '@axa-ch/materials/icons/material-design/expand_more.svg'
 
 import { classMap } from 'lit/directives/class-map.js';
 import { defineVersioned } from '../../../utils/component-versioning';
-import applyDefaults from '../../../utils/apply-defaults';
 import { sanitizeSVG } from '../../../utils/sanitize';
+import applyDefaults from '../../../utils/apply-defaults';
 import fireCustomEvent from '../../../utils/custom-event';
+import createRefId from '../../../utils/create-ref-id';
 import styles from './index.scss';
 
+const collapseContent = el => {
+  // get the height of the element's inner content, regardless of its actual size
+  const contentHeight = el.scrollHeight;
+  // temporarily disable all css transitions
+  const savedTransition = el.style.transition;
+  el.style.transition = '';
+
+  // on the next frame (as soon as the previous style change has taken effect),
+  // explicitly set the element's height to its current pixel height, so we
+  // aren't transitioning out of 'auto'
+  requestAnimationFrame(() => {
+    el.style.height = `${contentHeight}px`;
+    el.style.transition = savedTransition;
+
+    // on the next frame (as soon as the previous style change has taken effect),
+    // have the element transition to height: 0
+    requestAnimationFrame(() => {
+      el.style.height = '0';
+    });
+  });
+};
+
+const expandContent = el => {
+  // get the height of the element's inner content, regardless of its actual size
+  const contentHeight = el.scrollHeight;
+  // have the element transition to the height of its inner content
+  el.style.height = `${contentHeight}px`;
+
+  const expandAnimation = () => {
+    // remove "height" from the element's inline styles, so it can return to its initial value
+    el.style.height = null;
```

Comment:
question: when I read remove, I expected a `delete el.style.height;`. Would that not work here, and if not, why?